Mathlib/AlgebraicTopology/FundamentalGroupoid/Basic.lean

```diff
@@ -273,109 +248,62 @@ def FundamentalGroupoid (X : Type u) := X
 def fundamentalGroupoidFunctor : TopCat ⥤ CategoryTheory.Grpd where
   obj X := { α := FundamentalGroupoid X }
-  map f :=
-    { obj := f
-      map := fun {X Y} p => by exact Path.Homotopic.Quotient.mapFn p f
-      map_id := fun X => rfl
-      map_comp := fun {x y z} p q => by
-        refine Quotient.inductionOn₂ p q fun a b => ?_
-        simp only [comp_eq, ← Path.Homotopic.map_lift, ← Path.Homotopic.comp_lift, Path.map_trans]
-        -- This was not needed before leanprover/lean4#2644
-        erw [ ← Path.Homotopic.comp_lift]; rfl}
-  map_id X := by
-    simp only
-    change _ = (⟨_, _, _⟩ : FundamentalGroupoid X ⥤ FundamentalGroupoid X)
-    congr
-    ext x y p
-    refine' Quotient.inductionOn p fun q => _
-    rw [← Path.Homotopic.map_lift]
-    conv_rhs => rw [← q.map_id]
-  map_comp f g := by
-    simp only
-    congr
-    ext x y p
-    refine' Quotient.inductionOn p fun q => _
-    simp only [Quotient.map_mk, Path.map_map, Quotient.eq']
-    rfl
+  map := map
+  map_id X := by simp only [map]; congr; ext x y ⟨p⟩; rfl
+  map_comp f g := by simp only [map]; congr; ext x y ⟨p⟩; rfl
 #align fundamental_groupoid.fundamental_groupoid_functor FundamentalGroupoid.fundamentalGroupoidFunctor
 
+scoped notation "πₘ" => FundamentalGroupoid.map
 scoped notation "π" => FundamentalGroupoid.fundamentalGroupoidFunctor
-scoped notation "πₓ" => FundamentalGroupoid.fundamentalGroupoidFunctor.obj
-scoped notation "πₘ" => FundamentalGroupoid.fundamentalGroupoidFunctor.map
 
 theorem map_eq {X Y : TopCat} {x₀ x₁ : X} (f : C(X, Y)) (p : Path.Homotopic.Quotient x₀ x₁) :
     (πₘ f).map p = p.mapFn f := rfl
 #align fundamental_groupoid.map_eq FundamentalGroupoid.map_eq
 
-/-- Help the typechecker by converting a point in a groupoid back to a point in
-the underlying topological space. -/
-@[reducible]
-def toTop {X : TopCat} (x : πₓ X) : X := x
-#align fundamental_groupoid.to_top FundamentalGroupoid.toTop
-
-/-- Help the typechecker by converting a point in a topological space to a
-point in the fundamental groupoid of that space. -/
-@[reducible]
-def fromTop {X : TopCat} (x : X) : πₓ X := x
-#align fundamental_groupoid.from_top FundamentalGroupoid.fromTop
-
-/-- Help the typechecker by converting an arrow in the fundamental groupoid of
-a topological space back to a path in that space (i.e., `Path.Homotopic.Quotient`). -/
--- Porting note: Added `(X := X)` to the type.
-@[reducible]
-def toPath {X : TopCat} {x₀ x₁ : πₓ X} (p : x₀ ⟶ x₁) : Path.Homotopic.Quotient (X := X) x₀ x₁ := p
-#align fundamental_groupoid.to_path FundamentalGroupoid.toPath
-
-/-- Help the typechecker by converting a path in a topological space to an arrow in the
-fundamental groupoid of that space. -/
-@[reducible]
-def fromPath {X : TopCat} {x₀ x₁ : X} (p : Path.Homotopic.Quotient x₀ x₁) : x₀ ⟶ x₁ := p
-#align fundamental_groupoid.from_path FundamentalGroupoid.fromPath
+#noalign fundamental_groupoid.to_path
+#noalign fundamental_groupoid.from_path
```

Comment:
These were not type correct (`toPath` should say `Path.Homotopic.Quotient (X := X) (toTop x₀) (toTop x₁)` and `fromPath` should say `fromPath x₀ ⟶ fromPath x₁`), so I just remove them.